src/app/src/components/ModalSections/Introduction.js

```diff
@@ -0,0 +1,89 @@
+import { useState } from 'react';
+
+import {
+    Box,
+    Button,
+    Collapse,
+    Heading,
+    Text,
+    List,
+    ListIcon,
+    ListItem,
+} from '@chakra-ui/react';
+import {
+    CheckIcon,
+    ChevronDownIcon,
+    ChevronRightIcon,
+} from '@heroicons/react/solid';
+
+import ModalSection from './ModalSection';
+import { heroToChakraIcon } from '../../util/iconConverter';
+
+export default function Introduction({ goToNextSection }) {
+    const [showLearnMore, setShowLearnMore] = useState(false);
+
+    const LearnMoreArrow = heroToChakraIcon(
+        showLearnMore ? ChevronDownIcon : ChevronRightIcon
+    );
+
+    return (
+        <ModalSection
+            heading='Welcome to Boundary Sync'
+            nextButton={
+                <Button variant='cta' onClick={goToNextSection}>
+                    Get Started
+                </Button>
+            }
+        >
+            <Text>
```

Comment:
Let's add a [`textStyle`](https://chakra-ui.com/docs/styled-system/text-and-layer-styles#text-styles) here for this block:

```diff
diff --git a/src/app/src/components/ModalSections/Introduction.js b/src/app/src/components/ModalSections/Introduction.js
index bb6d28b..9cb87a7 100644
--- a/src/app/src/components/ModalSections/Introduction.js
+++ b/src/app/src/components/ModalSections/Introduction.js
@@ -35,7 +35,7 @@ export default function Introduction({ goToNextSection }) {
                 </Button>
             }
         >
-            <Text>
+            <Text textStyle='welcomeIntro'>
                 Build a digital map of your service area boundary in minutes for
                 free. No complex software or training needed.
             </Text>
diff --git a/src/app/src/theme.js b/src/app/src/theme.js
index 5a5af18..f555656 100644
--- a/src/app/src/theme.js
+++ b/src/app/src/theme.js
@@ -45,7 +45,14 @@ const theme = extendTheme({
     },
     fonts: {
         heading: `'Quicksand', sans-serif`,
-        body: `'Quicksand', sans-serif`,
+        body: `'Inter', sans-serif`,
+    },
+    textStyles: {
+        welcomeIntro: {
+            fontFamily: 'heading',
+            fontSize: 'xl',
+        },
+    },
     },
 });
 
```

Reply:
36534bf6c6c65872deb171434e8d77535ccd2ac3